datafusion/core/src/config.rs

```diff
@@ -119,6 +125,16 @@ impl BuiltInConfigs {
                 predicate push down.",
                 false,
             ),
+            ConfigDefinition::new_bool(
```

Comment:
can you run `./dev/update_config_docs.sh` to re-generate the documentation to include these new options

Reply:
The docs/source/user-guide/configs.md is updated. Thanks!